xdsl/dialects/func.py

```diff
@@ -28,6 +28,15 @@ class FuncOp(Operation):
     function_type = AttributeDef(FunctionType)
     sym_visibility = OptAttributeDef(StringAttr)
 
+    def verify_(self) -> None:
+        # TODO: how to verify that there is a terminator?
```

Comment:
IMO, this will require deeper changes in how regions are handled. For the moment, we can leave this as is.